spec/fixtures/api/isolated-preload.js

```diff
@@ -1,14 +1,11 @@
 // Ensure fetch works from isolated world origin
 fetch('http://localhost:1234')
 fetch('https://localhost:1234')
-fetch(`file://${__filename}`)
 
 const {ipcRenderer, webFrame} = require('electron')
 
 window.foo = 3
 
-webFrame.executeJavaScript('window.preloadExecuteJavaScriptProperty = 1234;')
```

Comment:
Instead of removing this, why not add `webFrame` to sandboxed renderer? A simple addition to https://github.com/electron/electron/blob/master/lib/sandboxed_renderer/api/exports/electron.js should do the trick.